util.lua

```diff
@@ -10,15 +10,21 @@ function djb2(s)
 	return hash
 end
 
-function hi(s) -- highlight
+function hi(s, mention) -- highlight
```

Comment:
I intended on adding a second argument to `hi()` already, for the kind of thing that is highlighted. Think `"nick"`, `"channel"`, `"mention"`.

To not mess with that, change the argument to `type`, add a `type = type or "nick"` down below, and instead of checking for `mention`, check for `type == "mention"`. Similarly, call `hi(conn.user, "mention")`.